### PR TITLE
Add basic validation to enclosure function

### PR DESCRIPTION
**Pull request by pfefferle**
/cc @akirk

Fixes Fatal errors if the `enclosure` meta does not match the core-format.

### CODE REVIEW

**Review by akirk — APPROVED** [review]
Thanks!

File: includes/functions.php [hunk 1]
```diff
@@ -1006,6 +1006,11 @@ function get_enclosures( $post_id ) {
 
 	$enclosures = array_map(
 		function ( $enclosure ) {
+			// Check if the enclosure is a string.
+			if ( ! $enclosure || ! is_string( $enclosure ) ) {
+				return false;
```
Comment (akirk) [inline includes/functions.php:1011]: Should this return `null` to remove it from the array?

**Review by akirk — CHANGES_REQUESTED** [review]
I think we should remove the invalid values using `array_filter()`

**Comment by pfefferle** [comment]
There is an `array_filter` and it also filters `false`.

File: includes/functions.php (hunk 1 above)
Comment (pfefferle) [reply includes/functions.php:1011]: Both will be filtered by `array_filter` but I am also fine with `null`.

**Review by akirk — APPROVED** [review]
Oops, sorry. I missed that!

**Pull request merged**